docs/setup/provision.md

```diff
@@ -0,0 +1,12 @@
+# Connect Create® 3 to Wi-Fi
+## The Basics
+Follow the main guide for getting started [here](https://edu.irobot.com/create3-setup).
+
+## Select RMW Implementation
+![Application Configuration Detail](data/appconfig.png)
+
+If you are planning to use ROS 2, make sure you have selected the correct RMW implementation.
+This can be found in Application &rarr; Configuration.
```

Comment:
Should we  mention that we  are talking about the webserver?

Reply:
Also suggestion:
 - `done from` rather than `found in` ?
 - add `as shown in the above picture` ?

Reply:
Will check in some edits soon.